SBD.GL.Win10/SBD.GL.Win10.csproj

```diff
@@ -82,61 +84,38 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Program.cs" />
+    <Compile Include="Program10.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="App.config" />
-  </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\SBD.GL.Win\SBD.GL.Win.csproj">
       <Project>{d05d93df-312d-4d4e-b980-726871ec7833}</Project>
       <Name>SBD.GL.Win</Name>
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="DevExpress.ExpressApp">
-      <Version>18.2.7</Version>
-    </PackageReference>
-    <PackageReference Include="DevExpress.ExpressApp.Core.All">
-      <Version>18.2.7</Version>
-    </PackageReference>
-    <PackageReference Include="DevExpress.ExpressApp.EF">
-      <Version>18.2.7</Version>
-    </PackageReference>
-    <PackageReference Include="DevExpress.Persistent.BaseImpl.EF">
-      <Version>18.2.7</Version>
-    </PackageReference>
-    <PackageReference Include="EntityFramework">
-      <Version>6.2.0</Version>
-    </PackageReference>
-    <PackageReference Include="SQLite.CodeFirst">
-      <Version>1.5.1.25</Version>
-    </PackageReference>
-    <PackageReference Include="System.Data.SQLite">
-      <Version>1.0.109.2</Version>
-    </PackageReference>
-    <PackageReference Include="System.Data.SQLite.Core">
-      <Version>1.0.109.2</Version>
-    </PackageReference>
-    <PackageReference Include="System.Data.SQLite.EF6">
-      <Version>1.0.109</Version>
-    </PackageReference>
-    <PackageReference Include="System.Data.SQLite.Linq">
-      <Version>1.0.109</Version>
-    </PackageReference>
+    <None Include="..\SBD.GL.Win\App.config">
+      <Link>App.config</Link>
+    </None>
   </ItemGroup>
   <ItemGroup>
-    <BootstrapperPackage Include=".NETFramework,Version=v4.7.2">
-      <Visible>False</Visible>
-      <ProductName>Microsoft .NET Framework 4.7.2 %28x86 and x64%29</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
+    <Content Include="..\SBD.GL.Win\bin\$(Platform)\$(Configuration)\Model.Cache.xafml">
```

Comment:
Here I assume that the output directory is $(Platform)\$(Configuration).
To make it work in AnyCpu we could use an [MSBuild Condition](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2019) and check if Configuration is AnyCPU and change the path to `..\SBD.GL.Win\bin\$(Platform)\Model.Cache.xafml` instead